### ISSUE TITLE
1.3 steam - sub priority is reset on window close

### ISSUE THREAD

**Issue by ttr** [fluffy-mods/WorkTab#186]
**Describe the bug**  
when selecting sub priority of a task, after window close it being reset to priority of main work

**Steps to reproduce**  
expand any main work (eg construction) change sub priority to different value (eg, smoothing floors to 0, while main is 2), close work tab and re-open it - sub priority is reset to value of main.
Changing main does work as expected

**Other mods**  
about 250 mods - i can provide a list if needed.

**Output log**  
https://gist.github.com/3bc4dedd8cbb4adc328aed14cba34722

**Save game**  
https://www.dropbox.com/s/qcdqv57ouf1ipbr/Oinkers.rws?dl=0

**Environment**  
 - OS: win 10
 - 64bit
 
**Game**
1.3, steam, 64b

**Comment by jolieber** [comment]
This happened to me too. It was working all the way until I arrested one of my own pawns, and now it won't work,

**Comment by derplordofdeath** [comment]
This is happening to me right now i dont remember it being a problem up until recently

**Comment by Domarovk** [comment]
Also having the same issue as of new save created today with new mods

**Comment by Wolfsblvt** [comment]
Yup, happening for me as well. Can't set butcher priority lower than the rest of the cooking. Worked in the past without any issues, can't get it to work in 1.3 sadly.
